ykrt/src/compile/jitc_yk/codegen/abs_stack.rs

```diff
@@ -0,0 +1,81 @@
+//! The abstract stack.
+
+/// This data structure keeps track of an abstract stack pointer for a JIT frame during code
+/// generation. The abstract stack pointer is zero-based, so the stack pointer value also serves as
+/// the size of the stack.
+///
+/// The implementation is platform agnostic: as the stack gets bigger, the stack pointer grows
```

Comment:
s/the stack pointer/abstract stack pointer/

Reply:
Fixed in https://github.com/ykjit/yk/pull/971/commits/6527226c2f1d6f3958634ca168bcc435395057d9